app/commands/get_user_review_request_metrics.rb

```diff
@@ -35,7 +36,7 @@ def add_pr_data_to_hash(pull_request)
       unless time_delta > 1.week
         @entry_count += 1
         @total_time += time_delta
-        @review_requests_dic[pull_request.id] = get_pr_info(pull_request, review_req, time_delta)
+        @review_requests_dic["pull_requests_information"][pull_request.id] = get_pr_info(pull_request, review_req, time_delta)
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Layout/LineLength: Line is too long. [126/100]